casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -127,7 +127,8 @@ final case class CasperShardConf(
     bondMinimum: Long,
     bondMaximum: Long,
     epochLength: Int,
-    quarantineLength: Int
+    quarantineLength: Int,
+    minPhloPrice: Int
```

Comment:
PhloPrice on DeployData is Long so better to have Long here also. Sorry, my mistake telling you Int.

Reply:
Done. I couldn't find a way, how to change `int32` to `int64` in [DeployServiceCommon.proto](https://github.com/stanislavlyalin/rchain/blob/1953cd52c4b8247be7b9ddf77b5fa774a742a93a/models/src/main/protobuf/DeployServiceCommon.proto). Changing type leads to error in generated code. So additional converting in gRPC `status` method [was provided](https://github.com/stanislavlyalin/rchain/blob/82ce6fb49eb168e423962714b4ec580c87fbb36f/node/src/main/scala/coop/rchain/node/api/DeployGrpcServiceV1.scala#L312).

Reply:
Fix is in this commit https://github.com/rchain/rchain/pull/3556/commits/6d4a87a8e022253cb8fbc9de51a22f63156b8137. The reason for the error was in custom implementation for object hashCode which must enumerate all types that are mentioned in generated code with [StacksafeScalapbGenerator](https://github.com/rchain/rchain/blob/af703b79cd27/project/StacksafeScalapbGenerator.scala).